addons/website_sale/models/product.py

```diff
@@ -191,14 +191,15 @@ def _website_price(self):
         qty = self._context.get('quantity', 1.0)
         partner = self.env.user.partner_id
         pricelist = self.env['website'].get_current_website().get_current_pricelist()
+        company_id = self.env['website'].get_current_website().company_id
```

Comment:
As you are using `get_current_website` 2 times, store its value in an intermediate variable.